packages/types/lib/sync/api.ts

```diff
@@ -36,6 +36,21 @@ export type PostSyncVariant = Endpoint<{
     Success: { id: string; name: string; variant: string };
 }>;
 
+export type UpdateSyncConnectionFrequency = Endpoint<{
```

Comment:
I would have the name of the consistent with the endpoint path. wdyt? 
```suggestion
export type UpdateConnectionFrequency = Endpoint<{
```